src/Testcontainers/Clients/TestcontainersClient.cs

```diff
@@ -212,7 +215,7 @@ await tarOutputStream.WriteAsync(fileContent, 0, fileContent.Length, ct)
 
         memStream.Seek(0, SeekOrigin.Begin);
 
-        await this.containers.ExtractArchiveToContainerAsync(id, "/", memStream, ct)
+        await this.containers.ExtractArchiveToContainerAsync(id, Path.AltDirectorySeparatorChar.ToString(), memStream, ct)
```

Comment:
Quite a lot of [docs](https://learn.microsoft.com/en-us/dotnet/api/system.io.path.altdirectoryseparatorchar?view=net-6.0), just to explain
> The value of this field is a slash ('/') on both Windows and Unix-based operating systems.

😅